java/api/src/main/java/org/ray/api/RuntimeContext.java

```diff
@@ -0,0 +1,46 @@
+package org.ray.api;
+
+import org.ray.api.id.UniqueId;
+
+/**
+ * A class used for getting information of Ray runtime.
+ */
+public interface RuntimeContext {
+
+  /**
+   * Get the current Driver ID.
+   *
+   * If called in a driver, this returns the driver ID. If called in a worker, this returns the ID
+   * of the associated driver.
+   */
+  UniqueId getCurrentDriverId();
+
+  /**
+   * Get the current actor ID.
+   *
+   * Note, this can only be called in actors.
+   */
+  UniqueId getCurrentActorId();
+
+  /**
+   * Returns true if the current actor was reconstructed, false if it's created for the first time.
+   *
+   * Note, this method should only be called from an actor creation task.
+   */
+  boolean wasCurrentActorReconstructed();
```

Comment:
Let's add a check to make sure this method is called from actor creation task. Also, we can test it in `ActorReconstructionTest::testActorReconstruction`